model/github_runner.rb

```diff
@@ -25,6 +25,15 @@ def runner_url
     "http://github.com/#{repository_name}/settings/actions/runners/#{runner_id}" if runner_id
   end
 
+  def display_state
+    return vm.display_state if vm
+    if strand
+      return "deleted" if strand.label == "wait_vm_destroy"
+      return "reached_concurrency_limit" if strand.label == "wait_concurrency_limit"
+    end
+    "not_created"
```

Comment:
```suggestion 
    case strand&.label
    when "wait_vm_destroy"
      "deleted"
    when "wait_concurrency_limit"
      "reached_concurrency_limit"
    else
      "not_created"
    end
```

It blows up the line count, but probably has fewer expressions and definitely less flow control. Not entirely coincidentally, it's half the width.

Reply:
Another alternative which doesn't blow up line counts

```
    case strand&.label
    when "wait_vm_destroy" then "deleted"
    when "wait_concurrency_limit" then "reached_concurrency_limit"
    else "not_created"
    end
```

Reply:
Sure :)